cmd/genji/shell/command.go

```diff
@@ -52,43 +52,50 @@ func runTablesCmd(db *genji.DB, cmd []string) error {
 
 // displayTableIndex prints all indexes that the given table contains.
 func displayTableIndex(db *genji.DB, tableName string) error {
-	err := db.View(func(tx *genji.Tx) error {
-		t, err := tx.GetTable(tableName)
-		if err != nil {
-			return err
-		}
+	ctx := context.Background()
+	res, err := db.Query(ctx, "SELECT * FROM __genji_indexes")
```

Comment:
Why not use SQL for the filtering as well (i.e. `WHERE`)?

Reply:
`db.QueryDocument` I didn't know this function. 
About SQL it because sometimes I think weirdly. Change it.